module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -650,8 +650,8 @@ namespace platform {
                     /************************************************
                      *                  CENTRE OF PRESSURE          *
                      ************************************************/
-                    sensors->centreOfPressure = convert<double, 3>(
-                        utility::motion::kinematics::calculateCentreOfPressure(kinematicsModel, *sensors));
+                    // sensors->centreOfPressure = convert<double, 3>(
+                    //    utility::motion::kinematics::calculateCentreOfPressure(kinematicsModel, *sensors));
```

Comment:
Yeah, given that we have memory corruption issues, just because this function segfaults doesn't mean it's the thing causing it. So we should work out why this function is segfaulting before removing it.